<launch>
    <arg name="robot_namespace" default="/capra/signal_strength" />

    <node name="wifi_mapper_node" pkg="capra_signal_strength" type="wifi_mapper_node" ns="$(arg robot_namespace)" output="screen"/>
</launch>